easybuild/framework/easyconfig/types.py

```diff
@@ -116,3 +110,29 @@ def convert_value_type(val, typ):
         raise EasyBuildError("No conversion function available (yet) for target type %s", typ)
 
     return res
+
+def to_toolchain(tcspec):
+    """
+    Convert a toolchain string "foo, v0" to a dictionary {'name':'foo', 'version':'v0'}
```

Comment:
use an actual toolchain as example, e.g. `intel, 2015a` and `{'name': 'intel', 'version': '2015a'}`